reporter-config3/pom.xml

```diff
@@ -93,6 +105,14 @@
             <artifactId>simpleclient_servlet</artifactId>
             <optional>true</optional>
         </dependency>
+        <!-- kafka -->
+	    <dependency>
+	      <groupId>com.github.mstump</groupId>
+	      <artifactId>metrics-kafka</artifactId>
+          <version>c804bf1874</version>
```

Comment:
Any chance you are planning a 0.0.1 release?  https://github.com/mstump/metrics-kafka/commit/47ff7218736c7a6ce572d119a760b4f983b9d81c